### PR TITLE
Example and docs for Python span profiles

### PR DESCRIPTION
**Pull request by aleks-p**
- enables OpenTelemetry based tracing in the Python ride share Flask example
- adds the Python ride share Flask app to examples/tracing/tempo, to demonstrate span profiles in Python
- adds docs for Python span profiles

### CODE REVIEW

File: docs/sources/configure-client/trace-span-profiles/_index.md [hunk 1]
```diff
@@ -21,11 +21,12 @@ Key benefits and features:
 Get started:
 
 - Configure Pyroscope: Begin sending profiling data to unlock the full potential of Span Profiles
-- Client-Side Packages: Easily link traces and profiles using available packages for Go, Java, Ruby and .NET
+- Client-Side Packages: Easily link traces and profiles using available packages for Go, Java, Ruby, .NET and Python
```
Comment (knylander-grafana) [inline docs/sources/configure-client/trace-span-profiles/_index.md:24]: We use sentence case in our documentation. Not everything is consistent yet, but we're working on it.
```suggestion
- Client-Side packages: Easily link traces and profiles using available packages for Go, Java, Ruby, .NET and Python
```

File: docs/sources/configure-client/trace-span-profiles/dotnet-span-profiles.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ To use Span Profiles, you need to:
 Your applications must be instrumented for profiling and tracing before you can use span profiles.
 
 * Profiling: Your application must be instrumented with Pyroscope's .NET instrumentation library. Refer to the [.NET]({{< relref "../language-sdks/dotnet" >}}) guide for instructions.
-* Tracing: Your application must be instrumented with OpenTelemetry traces. Refer to the [OpenTelemetry](https://opentelemetry.io/docs/net/getting-started/) guide for isntructions.
+* Tracing: Your application must be instrumented with OpenTelemetry traces. Refer to the [OpenTelemetry](https://opentelemetry.io/docs/languages/net/getting-started/) guide for isntructions.
```
Comment (knylander-grafana) [inline docs/sources/configure-client/trace-span-profiles/dotnet-span-profiles.md:39]: Thank you for updating the links!

**Review by knylander-grafana — COMMENTED** [review]
Thank you for updating the docs! The updates look good. I'm leaving a comment only, since I can't approve the other code changes.

**Comment by knylander-grafana** [comment]
Do you want to backport this PR to 1.5 so it will be published before the next release?

File: docs/sources/configure-client/trace-span-profiles/_index.md [hunk 3]
```diff
@@ -21,11 +21,12 @@ Key benefits and features:
 Get started:
 
 - Configure Pyroscope: Begin sending profiling data to unlock the full potential of Span Profiles
-- Client-Side Packages: Easily link traces and profiles using available packages for Go, Java, Ruby and .NET
+- Client-side packages: Easily link traces and profiles using available packages for Go, Java, Ruby, .NET and Python
```
Comment (kolesnikovae) [inline docs/sources/configure-client/trace-span-profiles/_index.md:24]:
```suggestion
- Client-side packages: Easily link traces and profiles using available packages for Go, Java, Ruby, .NET, and Python
```

**Review by kolesnikovae — APPROVED** [review]

**Pull request merged**